crates/orderbook/src/fee.rs

```diff
@@ -361,6 +368,12 @@ impl MinFeeCalculating for MinFeeCalculator {
             (subsidized_min_fee, fee_data.sell_token),
         );
 
+        if !self.store_computed_fees {
+            // Set an expired timeout to signal that this fee estimate is only indicative
+            // and not supposed to be used to create actual orders with it.
+            official_valid_until = DateTime::from_utc(NaiveDateTime::from_timestamp(0, 0), Utc);
+        }
```

Comment:
I'm curious why this branch was placed here instead of in the `else` clause ☝️. They have slightly different semantics (here, all `fast` quotes have invalid fees, in the `else` clause only "fresh" `fast` quotes have invalid fees).

I'm not sure which is better:
- If its always 0 then its consistent - never use `fast` for real fees
- If its only 0 for fresh quotes, then we know what the official fee will be, so you can use `fast` sometimes.

Just wondering if you thought about it. I'm fine either way (slightly preferring consistent behaviour with `fast` estimates - so as is here) but it would be good to also document that here, so the expected outcome isn't forgotten.

Reply:
I initially had the code in the `else` branch you highlighted but decided to move it before the final return to make the `fast` response consistent as you pointed out.
I think since our exchange isn't fast paced anyway end users would not have any advantage of having the chance that the `fast` price be **might** usable **sometimes**. With that in mind having 100% consistency seems more desirable to me.
>it would be good to also document that decision here

We can even document it in `openapi.yaml` to make it public that `fast` prices are no good for actual orders.
(I guess I will do both)

Reply:
> With that in mind having 100% consistency seems more desirable to me.

Agreed, just wanted to make sure it was a conscious decision and not a per-chance thing 😛.

> We can even document it in openapi.yaml

Yes! That would be awesome as well!